src/bugmon_tc/process/cli.py

```diff
@@ -107,7 +107,7 @@ def parse_args(argv: Any = None) -> argparse.Namespace:
     return args
 
 
-def main(argv: Optional[Dict[str, Any]] = None) -> None:
+def main(argv: Any = None) -> None:
```

Comment:
This is odd. Was it causing problems? `Optional[Dict[str, str]]` would be the correct type.

Reply:
Fixed via [ae9ff1b](https://github.com/MozillaSecurity/bugmon-tc/pull/15/commits/ae9ff1bc7a5d86dde7e3e240e1cbf11aac406182).